src/client/components/App/App.jsx

```diff
@@ -1,26 +1,47 @@
-import styles from './styles.scss';
-
 import React, { Component } from 'react';
-import Article from '../Article/Article.jsx';
-import Summary from '../Summary/Summary.jsx';
-import Title from '../Title/Title.jsx';
-import PostScriptum from '../PostScriptum/PostScriptum.jsx';
+import { moviesList } from './../mockMoviesList';
+
+import ErorrBoundary from '../ErrorBoundary/index.jsx';
+import Search from '../Search/index.jsx';
```

Comment:
можно не писать `index.jsx` в пути, вебпак сам поймёт, что нужно пойти в index:

```suggestion
import Search from '../Search';
```